gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/GradebookNgBusinessService.java

```diff
@@ -120,6 +121,8 @@
 
 	public static final String ASSIGNMENT_ORDER_PROP = "gbng_assignment_order";
 
+	private Collator collator = Collator.getInstance();
```

Comment:
This could probably go in each comparator but I am going to split these out into their own class eventually so I'll do this then,